client/app/bundles/course/survey/actions/questions.js

```diff
@@ -77,10 +76,9 @@ export function finalizeOrder(successMessage, failureMessage, onError) {
     const {
       surveysFlags: { isQuestionMoved },
       surveys,
-    } = getState();
-    if (!isQuestionMoved) {
-      return;
-    }
+    } = getState().surveys;
```

Comment:
Have you checked getState() in other files as well?

Reply:
Yes.